pins.go

```diff
@@ -0,0 +1,9 @@
+package drivers
+
+// PinOutput is hardware abstraction for a pin which outputs a
+// digital signal (high or low voltage).
```

Comment:
```suggestion
// digital signal (high or low level).
```
What I have found with the search engine: "Low voltage ranges from 0 to 50 volts, while high voltage ranges from 1,000 to 500,000 volts."